frontend/jupyter/src/app/pages/form/form-new/form-advanced-options/form-advanced-options.component.html

```diff
@@ -1,10 +1,4 @@
-<lib-form-section 
-  title="Miscellaneous Settings" 
-  i18n-title
-  text="Other possible settings to be applied to the Notebook Server."
-  i18n-text
-  icon="fa:fas:cogs"
->
+<lib-form-section title="Miscellaneous Settings" i18n-title>
```

Comment:
Comment: I am sad that they removed the icons. I will add a comment in the epic about this, since I think they should be kept.